modules/ruby.bash

```diff
@@ -1,25 +1,21 @@
 #!/usr/bin/env bash
 
+set -e
+
 osx_bootstrap="$(cd "$(dirname "$0")/.." && pwd -P)"
 source "$osx_bootstrap/modules/functions.bash"
 
-info_echo "Enable rbenv alias"
-eval "$(rbenv init -)"
-
-info_echo "Set default gems list"
-echo "bundler" >> "$(brew --prefix rbenv)/default-gems"
-
-info_echo "Installing latest stable Ruby..."
-ruby_version="$(rbenv install -l | grep -E "^\\s*([.0-9]+)$" | tail -1)"
-
-if test -z "$(rbenv versions --bare|grep $ruby_version)"; then
-  info_echo "Install Ruby $ruby_version"
-  rbenv install $ruby_version
+if ! command -v rbenv &> /dev/null
+then
+  echo 'Installing rbenv'
+  # shellcheck disable=SC2016
+  [ -f "$HOME/.zprofile" ] && echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> ~/.zprofile
```

Comment:
What is this line? I don't think it is related to `ruby.bash` file

Reply:
This sc disable only for single quotes, or here can be some errors?

Reply:
@DenisZackharov  I talk about `/opt/homebrew/bin/brew shellenv`

Reply:
Do you mean that this line need move in brew.bash file?